### PR TITLE
[sidebar] Including Sidebar Library in Repository

### PR DESCRIPTION
**Pull request by rpao**
- The sidebar must work as a template where can be added content to the header, body, and footer.
- A server controls the sidebars' actions, including opening and closing, using the sidebar's id.
- The main UI/UX designs are the same as the original code.

### CODE REVIEW

File: projects/lib-sidebar/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "lib-sidebar",
+  "version": "0.0.1",
+  "peerDependencies": {
+    "@angular/common": "^11.0.9",
```
Comment (rodrigoperazzo) [inline projects/lib-sidebar/package.json]: Can we try to make the library compatible with Angular versions from 8+?

File: projects/lib-sidebar/src/lib/service/lib-sidebar.service.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  providedIn: 'root'
+})
+
+export class LibSidebarService {
+
+  private sidebars: any[] = [];
+
+  constructor() { }  
+
+  public add(sidebar: any): boolean {
```
Comment (CGSILVA) [inline projects/lib-sidebar/src/lib/service/lib-sidebar.service.ts]: Please create a type for siderbar

File: projects/lib-sidebar/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# LibSidebar
+
+This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.0.9.
```
Comment (CGSILVA) [inline projects/lib-sidebar/README.md]: If you change Angular version remember to change here

**Review by CGSILVA — CHANGES_REQUESTED** [review]

File: projects/lib-sidebar/src/lib/service/lib-sidebar.service.ts (hunk 2 above)
Comment (rpao) [reply projects/lib-sidebar/src/lib/service/lib-sidebar.service.ts]: The sidebar receives an instance of LibSidebarComponent, so it was defined as type LibSidebarComponent.

File: projects/lib-sidebar/README.md (hunk 3 above)
Comment (rpao) [reply projects/lib-sidebar/README.md]: Angular Version is now the same as the Tooltip Directive.

File: projects/lib-sidebar/package.json (hunk 1 above)
Comment (rpao) [reply projects/lib-sidebar/package.json]: Angular Version is now the same as the Tooltip Directive.

**Pull request merged**